.prettierrc.yml

```diff
@@ -1,7 +1,3 @@
 trailingComma: es5
 semi: false
 singleQuote: true
-overrides:
```

Comment:
We should migrate to @oacore/configs as soon as possible. I am still waiting for your review of config and then I will create pull request

Reply:
I know. This is the only place we have Stylelint config. Let's start from this place and see how it works. I do not want often update @oacore/configs because something does not fit our workflow. It means many releases with big loop.

Reply:
This one is prettier config. We can postpone moving stylelint. I would definitely like to see it in configs one day for sure. :)

Reply:
However I still insist on using prettier config from @oacore/configs.

Reply:
Yes, I missed this. Sure. Feel free to push this update. I did not even notice that prettier config is local.

Reply:
I mean, we can replace it later